lib/Chat/ChatManager.php

```diff
@@ -521,6 +521,8 @@ public function deleteMessage(Room $chat, IComment $comment, Participant $partic
 
 		$this->referenceManager->invalidateCache($chat->getToken());
 
+		$this->unreadCountCache->clear($chat->getId() . '-');
```

Comment:
Seems like we just had to bust the unread cache on the delete message action. I checked to see if there will be any regression issues due to this seems like no.

As `true` is passed over here -

https://github.com/nextcloud/spreed/pull/13355/files#diff-3f8970573742fb41e3bee92a0f7b7fae33e60043734bcd1171095991c031a6aeR535

therefore the cache is not bust in the `addSystemMessage` function.